pkg/model/utils.go

```diff
@@ -17,22 +17,70 @@ package model
 
 import (
 	"github.com/jhump/protoreflect/dynamic"
+	"log"
+	"os"
 )
 
+var error = log.New(os.Stderr, "ERROR: ", 0)
```

Comment:
No need to define this variable. Just use errors.New() method to create and return the error